### PR TITLE
Mesh renderer(s) I18N

### PR DESCRIPTION
**Pull request by shrinktofit**
Re: #

### Changelog

*

Known problem: the `Materials` field is not translated.

zh:

image

en:

image

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/i18n/en/rendering.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                        displayName: 'Shadow Normal Bias',
+                        tooltip: 'Bias value (world space unit) that can avoid moire artifacts with surfaces that parallel to the directional light.',
+                    },
+                    'bakeSettings': {
+                        displayName: 'Bake Settings',
+                        tooltip: 'Bake settings related to global lighting.',
+                    },
+                    'isGlobalStandardSkinObject': {
+                        displayName: 'Global Standard Skin Object',
+                        tooltip: '(FIXME!!)Bias value (world space unit) that ensure globally unique standard skin model.',
```
Comment (shrinktofit) [inline editor/i18n/en/rendering.js]: @troublemaker52025  This tooltip is moved from previous version and seems incorrect. Please suggest me a change.

Comment (troublemaker52025) [reply editor/i18n/en/rendering.js:82]: Is the name ambiguous or is there some other reason to change it?

Comment (troublemaker52025) [reply editor/i18n/en/rendering.js]: “The engine needs to know the scale of the model corresponding to the skin material in order to calculate the skin scattered light correctly. If no model is checked, the model with the skin material will be automatically selected for calculation.”

**Review by arsen2010 — APPROVED** [review]

File: editor/i18n/en/rendering.js (hunk 1 above)
Comment (shrinktofit) [inline editor/i18n/en/rendering.js]:
```suggestion
                        tooltip: 'The engine needs to know the scale of the model corresponding to the skin material in order to calculate the skin scattered light correctly. If no model is checked, the model with the skin material will be automatically selected for calculation.',
```

**Review by troublemaker52025 — APPROVED** [review]

**Review by minggo — APPROVED** [review]

File: editor/i18n/en/modules/rendering.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                    },
+                    'bakeToReflectionProbe': {
+                        displayName: 'Bake To Reflection Probe',
+                    },
+                },
+            },
+            'MeshRenderer': {
+                groups: {
+                    DynamicShadow: {
+                        displayName: 'Dynamic Show Settings',
```
Comment (SantyWang) [inline editor/i18n/en/modules/rendering.js]:
```suggestion
                        displayName: 'Dynamic Shadow Settings',
```

Comment (shrinktofit) [inline editor/i18n/en/modules/rendering.js]:
```suggestion
                        displayName: 'Dynamic Shadow Settings',
```

**Review by SantyWang — APPROVED** [review]

**Pull request merged**